scripts/src/java/org/oppia/android/scripts/testing/TestBazelWorkspace.kt

```diff
@@ -67,10 +67,10 @@ class TestBazelWorkspace(private val temporaryRootFolder: TemporaryFolder) {
       createLibrary("${testName}Dependency")
     } else null to listOf()
     val buildFile = if (subpackage != null) {
-      if (!File(temporaryRootFolder.root, subpackage).exists()) {
-        temporaryRootFolder.newFolder(subpackage)
+      if (!File(temporaryRootFolder.root, subpackage.replace(".", "/")).exists()) {
```

Comment:
Why are these changes duplicated from your other PR?

Reply:
Ah, you reverted right before I submitted this comment! Marking as resolved.

Reply:
My bad I didn't see that, 
Reverted now